store/views.py

```diff
@@ -62,8 +61,19 @@ def loanBookView(request):
     If yes, then set the message to 'success', otherwise 'failure'
     '''
     # START YOUR CODE HERE
-    book_id = None # get the book id from post data
+    book_id = request.POST.get('bid')
+    book = Book.objects.get(id = book_id)
 
+    available_copies = BookCopy.objects.filter(book=book, status=True).all()
+    if (len(available_copies) > 0) :
+        response_data['message'] = "success"
+        issued = available_copies[0]
+        issued.status = False
+        issued.borrow_date = datetime.now()
+        issued.borrower = request.user
+        issued.save();
+    else :
+        response_data['message'] = "failure"
```

Comment:
You can return the specific message that no copy of the book is available, or disable the button in the frontend when the number of available books is 0.